.github/workflows/selftest.yml

```diff
@@ -35,6 +35,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install graphviz wheel pylint
+        pip install graphviz wheel flake8
```

Comment:
I think it would make sense to move `flake8` up to the line above

Reply:
Agree, fixed.